ui/src/customElements/FormSubmitCancel.ts

```diff
@@ -0,0 +1,4 @@
+import { createCustomElement } from '@/forms/custom-element'
+import FormSubmitCancel from '@/components/FormSubmitCancel.vue'
+
+createCustomElement('cc-form-submit-cancel')(FormSubmitCancel)
```

Comment:
Just to tell you,  one can declare them globally, like https://github.com/zazuko/cube-viewer/blob/489063e6d89f4c7ab2c02345184916ad73577d1a/vue.config.js#L32

But I prefer being explicit on it, like you did :)

Reply:
That's not the same. The line you linked is so that due does not treat custom elements as vue components.

Here it's registering the element with the browser itself

Reply:
Ok, I need to read more about that. ;)

Reply:
It wraps the native [`customElements.define()`](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry/define)